homeassistant/components/sensibo/binary_sensor.py

```diff
@@ -85,6 +85,13 @@ class SensiboDeviceBinarySensorEntityDescription(
         icon="mdi:motion-sensor",
         value_fn=lambda data: data.room_occupied,
     ),
+    SensiboDeviceBinarySensorEntityDescription(
+        key="filter_clean",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+        name="Filter Clean Required",
+        icon="mdi:timer",
+        value_fn=lambda data: data.filter_clean,
+    ),
```

Comment:
Since this is the same in both, you could make this a constant instead of writing it out twice